bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -44,6 +45,8 @@ pub struct ExecStep {
     pub bus_mapping_instance: Vec<OperationRef>,
     /// Error generated by this step
     pub error: Option<ExecError>,
+    /// Non existent account addresses
+    pub non_existent_accounts: HashSet<Address>,
```

Comment:
most probably do not need this, for circuit side, existing can be check by reading buss mapping record,  for `AccountFieldTag::NonExisting`, no need additional step info.

Reply:
Yes exactly 🙏, I fixed to get both `balance` and `exists` in circuit function `assign_exec_step` as below:
```
let (balance, exists) = match block.rws[step.rw_indices[5]] {
    Rw::Account {
        field_tag: AccountFieldTag::Balance,
        value,
        ..
    } => (
        RandomLinearCombination::random_linear_combine(
            value.to_le_bytes(),
            block.randomness,
        ),
        true,
    ),
    Rw::Account {
        field_tag: AccountFieldTag::NonExisting,
        ..
    } => (F::zero(), false),
    _ => unreachable!(),
};
```